packages/react-scripts/package.json

```diff
@@ -42,9 +42,9 @@
     "css-loader": "2.1.1",
     "dotenv": "6.2.0",
     "dotenv-expand": "4.2.0",
-    "eslint": "^6.1.0",
+    "eslint": "^5.3.0",
```

Comment:
eslint": "^6.1.0" is the latest one. Could we go for that?

Reply:
Airbnb config that we are based on does not yet support it :/

Reply:
Also, quite a lot of the other packages specify 5* as dependency still